core/src/test/java/io/kestra/core/docs/ClassPluginDocumentationTest.java

```diff
@@ -103,7 +103,7 @@ void trigger() throws URISyntaxException {
             PluginClassAndMetadata<AbstractTrigger> metadata = PluginClassAndMetadata.create(scan, Schedule.class, AbstractTrigger.class, null);
             ClassPluginDocumentation<? extends AbstractTrigger> doc = ClassPluginDocumentation.of(jsonSchemaGenerator, metadata, true);
 
-            assertThat(doc.getDefs().size()).isEqualTo(1);
```

Comment:
This is concerning now there is a lot of def in the Schedule.
I guess all conditions...